documentation/README.md

```diff
@@ -54,3 +54,36 @@ dokka {
   }
 }
 ```
+
+## For User Guide
```

Comment:
This is only relevant to JavaDoc projects, so maybe it should be moved as a sub-section under `For Doclava`?

Reply:
Moved under Doclava